x-pack/plugins/security/public/account_management/user_profile/user_profile.tsx

```diff
@@ -513,48 +513,46 @@ export const UserProfile: FunctionComponent<UserProfileProps> = ({ user, data })
           <EuiPageTemplate
             style={{ backgroundColor: euiTheme.colors.emptyShade }}
             className="eui-fullHeight"
-            pageContentBodyProps={{ style: { paddingLeft: 0, paddingRight: 0 } }}
             pageHeader={{
-              style: { paddingLeft: 0, paddingRight: 0 },
-              bottomBorder: true,
               pageTitle: (
                 <FormattedMessage
                   id="xpack.security.accountManagement.userProfile.title"
                   defaultMessage="Profile"
                 />
               ),
-              pageTitleProps: { id: titleId },
```

Comment:
__question:__ is there any reason why we need to remove this?

Reply:
Great catch - added back!